packages/augur-sdk/src/api/Gnosis.ts

```diff
@@ -379,7 +366,7 @@ export class Gnosis {
         shareTokenAddress,
         affiliates,
         fingerprint,
-        referralAddress,
+        NULL_ADDRESS,
```

Comment:
Style: I think having this named added value, since this is an otherwise an untyped tuple.